client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -1,7 +1,130 @@
 import { translate } from 'i18n-calypso';
-import type { AllowedTypes, SiteData, FormattedRowObj } from './types';
+import type {
+	AllowedTypes,
+	SiteData,
+	FormattedRowObj,
+	StatusEventNames,
+	ActionEventNames,
+	AllowedStatusTypes,
+	AllowedActionTypes,
+} from './types';
 import type { ReactChild } from 'react';
 
+// Event names for all actions for large screen(>960px) and small screen(<960px)
+export const actionEventNames: ActionEventNames = {
+	issue_license: {
+		large_screen: 'calypso_jetpack_agency_dashboard_issue_license_large_screen',
+		small_screen: 'calypso_jetpack_agency_dashboard_issue_license_small_screen',
+	},
+	view_activity: {
+		large_screen: 'calypso_jetpack_agency_dashboard_view_activity_large_screen',
+		small_screen: 'calypso_jetpack_agency_dashboard_view_activity_small_screen',
+	},
+	view_site: {
+		large_screen: 'calypso_jetpack_agency_dashboard_view_site_large_screen',
+		small_screen: 'calypso_jetpack_agency_dashboard_view_site_small_screen',
+	},
+	visit_wp_admin: {
+		large_screen: 'calypso_jetpack_agency_dashboard_visit_wp_admin_large_screen',
+		small_screen: 'calypso_jetpack_agency_dashboard_visit_wp_admin_small_screen',
+	},
+};
+
+// Returns event name based on the action type
+export const getActionEventNames = ( actionType: AllowedActionTypes, isLargeScreen: boolean ) => {
```

Comment:
As this function returns a single event name, I'd suggest using the singular form for its name.

```
export const getActionEventName = // ...
```